generate_openj9_scc.sh

```diff
@@ -21,47 +21,21 @@ SCC_SIZE="50m"
 # Runs for generating SCC
 SCC_GEN_RUNS_COUNT=3
 
-# is running on ubuntu or debian - package manager apt
-IS_APT_ENV=false
 # Intitialise default / allowed sample app status
-RUN_ECLIPSE=false
 RUN_TOMCAT=true
 
-# Intitialise default / required packages status
-INSTALL_GTK=false
-INSTALL_XVFB=false
+# sha512 checksums
+TOMCAT_CHECKSUM="0db27185d9fc3174f2c670f814df3dda8a008b89d1a38a5d96cbbe119767ebfb1cf0bce956b27954aee9be19c4a7b91f2579d967932207976322033a86075f98"
 
 # App download locations
-DOWNLOAD_PATH_ECLIPSE=/tmp/eclipse
 DOWNLOAD_PATH_TOMCAT=/tmp/tomcat
 
 # App installation locations
-INSTALL_PATH_ECLIPSE="${HOME}"/eclipse-home
 INSTALL_PATH_TOMCAT="${HOME}"/tomcat-home
```

Comment:
Please install it in `/opt` for consistency even though it will get deleted.

Reply:
Thanks for the review @dinogun , Made changes accordingly